.env.test

```diff
@@ -20,3 +20,8 @@ BROWSER_TESTS_USER_NAME=
 BROWSER_TESTS_USER_PASSWORD=
 BROWSER_TESTS_JWT_SIGN_SECRET=
 VITE_HELSINKI_PROFILE_URL=
+VITE_MATOMO_URL_BASE="//webanalytics.digiaiiris.com/js/"
```

Comment:
Same as I mentioned in the pipeline commit: This looks a broken URL to me when it's missing a protocol. I would include the protocol or leave the protocol-delimiters, the 2 slashes out. If this really is how the Matomo suggests us to do, then it's OK, but otherwise it jsut looks strange and broken.

Reply:
<img width="331" alt="image" src="https://github.com/user-attachments/assets/10fa766c-f67e-439c-b1ac-8a1fe029474d">
its just the value in script. if matomo knows how to use it, i dont think its a problem